docs/source/api/deliveryservices_sslkeys_generate_letsencrypt.rst

```diff
@@ -0,0 +1,64 @@
+..
+..
+.. Licensed under the Apache License, Version 2.0 (the "License");
+.. you may not use this file except in compliance with the License.
+.. You may obtain a copy of the License at
+..
+..     http://www.apache.org/licenses/LICENSE-2.0
+..
+.. Unless required by applicable law or agreed to in writing, software
+.. distributed under the License is distributed on an "AS IS" BASIS,
+.. WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+.. See the License for the specific language governing permissions and
+.. limitations under the License.
+..
+
+.. _to-api-deliveryservices-sslkeys-generate-letsencrypt:
+
+*************************************************
+``deliveryservices/sslkeys/generate/letsencrypt``
+*************************************************
+
+.. versionadded:: 1.4
+
+``POST``
+========
+Generates an SSL certificate and private key using Let's Encrypt for a :term:`Delivery Service`
+
+:Auth. Required: Yes
+:Roles Required: "admin" or "operations"
+:Response Type:  Object (string)
+
+Request Structure
+-----------------
+:key:             The :ref:`ds-xmlid` of the :term:`Delivery Service` for which keys will be generated
+:version:         An integer that defines the "version" of the key - which may be thought of as the sequential generation; that is, the higher the number the more recent the key
+:hostname:        The desired hostname of the :term:`Delivery Service`
+
+	.. note:: In most cases, this must be the same as the :ref:`ds-example-urls`
```

Comment:
nit: sentences should end with punctuation.

Field descriptions don't need that because they don't need to be complete sentences.

Reply:
done